src/store/zap/mutations.js

```diff
@@ -634,7 +649,10 @@ export function updateNotificationCount(state, value) {
 
 export function setDirtyState(state, isDirty) {
   if (state.isDirty != isDirty) {
-    state.isDirty = isDirty;
-    window.parent?.postMessage({eventId: 'dirty', eventData: { isDirty: isDirty }}, '*')
+    state.isDirty = isDirty
```

Comment:
Was this file formatted wrong, or are you not using the same formatter?

Reply:
Not sure, will have to check...